qlib/rl/order_execution/strategy.py

```diff
@@ -221,20 +490,11 @@ def __init__(
         if self._policy is not None:
             self._policy.eval()
 
-    def set_env(self, env: BaseEnvWrapper) -> None:
-        # TODO: This method is used to set EnvWrapper for interpreters since they rely on EnvWrapper.
-        # We should decompose the interpreters with EnvWrapper in the future and we should remove this method
-        # after that.
-
-        self._env = env
-        self._state_interpreter.env = self._action_interpreter.env = self._env
-
     def reset(self, outer_trade_decision: BaseTradeDecision = None, **kwargs: Any) -> None:
         super().reset(outer_trade_decision=outer_trade_decision, **kwargs)
 
-        # In backtest, env.reset() needs to be manually called since there is no outer trainer to call it
```

Comment:
Why can't we simply create a `DummyEnvWrapper` here?

Reply:
In my personal understanding, we don't need a `DummyEnvWrapper` at least for now.